homeassistant/components/camera/__init__.py

```diff
@@ -452,6 +400,12 @@ def unsub_track_time_interval(_event: Event) -> None:
         SERVICE_RECORD, CAMERA_SERVICE_RECORD, async_handle_record_service
     )
 
+    async def get_ice_servers() -> RTCIceServer:
+        # The following servers will replaced before the next stable release with
+        # STUN server provided by Home Assistant. Used Google ones for testing purposes.
+        return RTCIceServer(urls=["stun:stun.l.google.com:19302"])
```

Comment:
You can get the list of ice servers from `https://services.home-assistant.io/webrtc/ice_servers`.

Reply:
As discussed internally, we will merged it without it